src/reactblockui.d.ts

```diff
@@ -0,0 +1,33 @@
+import * as PropTypes from 'prop-types';
+
+export type BlockUiProps<T = {}> = React.HTMLAttributes<HTMLElement> & {
+    blocking?: PropTypes.bool,
+    keepInView?: PropTypes.bool,
+    children?: PropTypes.node,
+    renderChildren?: PropTypes.bool, // default to true
+    className?: PropTypes.string,
+    message?: PropTypes.oneOfType,
+    loader?: PropTypes.oneOfType,
+    tag?: PropTypes.oneOfType
```

Comment:
For now this works. In the future it would be better if we actually take the `props` declaration from the `tag` prop being passed in and `&` those props with this rather than letting any possible key/value pair in.